test/Faker/Provider/DateTimeTest.php

```diff
@@ -236,7 +236,7 @@ public function providerDateTimeInInterval()
 
     public function testFixedSeedWithMaximumTimestamp()
     {
-        $max = '2018-03-01 12:00:00';
+        $max = '2019-03-01 12:00:00';
```

Comment:
This was recently fixed, please remove it from the PR.

Reply:
@ppelgrims Thank you. I have changed it now.